FireMotD

```diff
@@ -1062,101 +1064,105 @@ $CP1$CP1$CP12$CP1H$C1R$CP1$CP1"
+        echo -e "$CP1        ${KS}Updates $ES ${VCL}$UpdateCount${VC} ${UpdateType} updates available."
     fi
     SessionCount=$(jq -r ".Sessions.Count" $ExportFile)
     if [[ ! -z $SessionCount ]] && [[ "$SessionCount" != "null" ]] ; then
-        echo -e "$CP1  ${KS}Sessions $ES ${VCL}$SessionCount"
+        echo -e "$CP1       ${KS}Sessions $ES ${VCL}$SessionCount"
     fi
     ProcessCount=$(jq -r ".Processes.Count" $ExportFile)
     ProcessMax=$(jq -r ".Processes.Max" $ExportFile)
     if [[ ! -z $ProcessCount ]] && [[ "$ProcessCount" != "null" ]] ; then
-        echo -e "$CP1 ${KS}Processes $ES ${VCL}$ProcessCount ${VC}running processes of ${VCL}$ProcessMax ${VC}maximum processes"
+        echo -e "$CP1      ${KS}Processes $ES ${VCL}$ProcessCount ${VC}running processes of ${VCL}$ProcessMax ${VC}maximum processes"
     fi
     HttpdVersion=$(jq -r ".Httpd.Version" $ExportFile)
     if [[ ! -z $HttpdVersion ]] && [[ "$HttpdVersion" != "null" ]] ; then
-        echo -e "$CP1    ${KS}Apache $ES ${VC}Version: ${VCL}$HttpdVersion"
+        echo -e "$CP1         ${KS}Apache $ES ${VC}Version: ${VCL}$HttpdVersion"
     fi
     NginxVersion=$(jq -r ".Nginx.Version" $ExportFile)
     if [[ ! -z $NginxVersion ]] && [[ "$NginxVersion" != "null" ]] ; then
-        echo -e "$CP1     ${KS}Nginx $ES ${VC}Version: ${VCL}$nginxVersion"
+        echo -e "$CP1          ${KS}Nginx $ES ${VC}Version: ${VCL}$nginxVersion"
     fi
     MysqlVersion=$(jq -r ".Mysql.Version" $ExportFile)
     MysqlDistribution=$(jq ".Mysql.Distribution" $ExportFile)
     if [[ ! -z $MysqlVersion ]] && [[ "$MysqlVersion" != "null" ]] ; then
-        echo -e "$CP1     ${KS}MySQL $ES ${VC}Version: ${VCL}$MysqlVersion ${VC}Distribution: ${VCL}$MysqlDistribution"
+        echo -e "$CP1          ${KS}MySQL $ES ${VC}Version: ${VCL}$MysqlVersion ${VC}Distribution: ${VCL}$MysqlDistribution"
     fi
     PostgresVersion=$(jq -r ".Postgres.Version" $ExportFile)
     if [[ ! -z $PostgresVersion ]] && [[ "$PostgresVersion" != "null" ]] ; then
-        echo -e "$CP1  ${KS}Postgres $ES ${VC}Version: ${VCL}$PostgreSqlVersion"
+        echo -e "$CP1       ${KS}Postgres $ES ${VC}Version: ${VCL}$PostgreSqlVersion"
     fi
     PhpVersion=$(jq -r ".Php.Version" $ExportFile)
     PhpMaxMemory=$(jq -r ".Php.MaxMemory" $ExportFile)
     if [[ ! -z $PhpVersion ]] && [[ "$PhpVersion" != "null" ]] ; then
-        echo -e "$CP1       ${KS}PHP $ES ${VC}Version: ${VCL}$PhpVersion, ${VC}Memory Limit: ${VCL}$PhpMaxMemory"
+        echo -e "$CP1            ${KS}PHP $ES ${VC}Version: ${VCL}$PhpVersion, ${VC}Memory Limit: ${VCL}$PhpMaxMemory"
+    fi
+    ElasticsearchVersion=$(jq -r ".Elasticsearch.Version" $ExportFile)
+    if [[ ! -z $ElasticsearchVersion ]] && [[ "$ElasticsearchVersion" != "Unknown" ]] && [[ "$ElasticsearchVersion" != "null" ]] ; then
+      echo -e "$CP1  ${KS}Elasticsearch $ES ${VC}Version: ${VCL}$ElasticsearchVersion"
```

Comment:
Sounds like new functionality (Elasticsearch integration?). Probably worth to separate to another commit.

Reply:
It already was in a previous commit for modern. No time to create separate commit for this. Going to merge.